README.md

```diff
@@ -15,17 +15,34 @@ module "rhythmic-iam-roles" {
 ```
 
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+## Requirements
+
+| Name | Version |
+|------|---------|
+| terraform | >= 0.12 |
```

Comment:
Did you actually test on `0.12.0` or just assume it works that far back?

Since this role is pretty much a purely internal role, I'd play it safe and set it to like `0.12.20` or something since we don't let our stuff get far behind.